samples/msal-node-samples/standalone-samples/README.md

```diff
@@ -20,23 +24,40 @@ HTTP:
 $ git clone https://github.com/AzureAD/microsoft-authentication-library-for-js.git
 ```
 
-Another option is to download the repository as a zip file by selecting "Download ZIP" from the root repository's dropdown "Code" menu. Once you've downloaded the ZIP file, you can decompress it locally and explore the code.
-
-Once you've successfully cloned the source code, which includes the samples, you can go through the pre-requisites list below in order to make sure you have all of the dependencies required to run these samples. 
+You can also download the repository as a zip file by selecting "Download ZIP" from the root repository's dropdown "Code" menu. Once you've downloaded the ZIP file, you can decompress it locally and explore the code.
 
 ### Pre-requisites
-- Ensure [all pre-requisites](../../../lib/msal-node/README.md#prerequisites) have been completed to run msal-node.
-- Install Node.js if needed (https://nodejs.org/en/).
-- Build the `msal-node` project with instructions provided in the [`README.md`](../../../lib/msal-node/README.md) or using the command `npm run build:package`.
+- By using MSAL Node, you are working with the Microsoft Identity ecosystem.  Read about [App Registration](https://docs.microsoft.com/en-us/graph/auth-register-app-v2) and register one for use with this code.
+- Install [Node.js](https://nodejs.org/en/) if needed
+- Install the MSAL Node package:  
+```bash
+npm install @azure/msal-node
+```
+- If you are customizing or building locally:
+```bash
+npm run build:package
+```
 
-Once you've completed all the pre-requisites, you can navigate to any of the sample application directories under `standalone-samples` and either execute each sample as-is, or customize it to use your own AzureAD app registration.
+Next, we navigate to the sample application directories under `standalone-samples` and either execute each sample as-is, or customize it to use your own [App Registration](https://docs.microsoft.com/en-us/graph/auth-register-app-v2).
 
-From the repository's root directory, navigate to a sample application (for example, the auth-code sample):
+From the repository's root directory, navigate to a sample application:
 
 ```bash
 $ cd samples/msal-node-samples/standalone-samples/auth-code
 ```
 
+### Which sample should I review?
+
+Review our [scenario docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-flows-app-scenarios) to pick a sample. 
+
+Continue to the appropriate sample to learn how it works, or to build it in to your existing app.
+
+- [Auth Code](auth-code/readme.md)  [web app]
```

Comment:
Putting these in a table might be a good idea, columns can be "Sample" and "Usage Scenario" or something like that.

Reply:
I thought of that, but hit a roadblock.  Not all flows are 1:1 with the terminology that an app developer may be thinking.  Using a hint like this doesn't say "this is what it is used for" rather, it is just a hint.

So, this is purposefully just a weaker "hint" statement rather than something I would put into a table.